addons/account-settings-capitalize/addon.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Fix capitalization of Account Settings",
+  "name": "Fixed capitalization of Account Settings",
```

Comment:
The word "fixed" usually means "static"; maybe `Capitalized "Account settings"`? (Doesn't matter for me, it's "an addon that does nothing" nonetheless)